auto_rx/config_reader.py

```diff
@@ -47,8 +48,8 @@ def read_auto_rx_config(filename):
 		config = ConfigParser.RawConfigParser()
```

Comment:
Should we change this to `config = ConfigParser.RawConfigParser(auto_rx_config)` so that it uses the defaults if a config option is missing from the config file?